modules/authentication/proc-enabling-authentication-with-rhbk.adoc

```diff
@@ -35,7 +35,7 @@ Save the value for the next step:
 +
 --
 .`{my-app-config-file}` fragment with mandatory fields to enable authentication with {rhbk}
-[source,yaml]
+[source,yaml,subs="+attributes,+quotes"]
```

Comment:
Consider omitting this substitutions statement, which is confusing and misleading:

* In this example, if you enable the `attributes` substitutions, you assert that `{AUTH_OIDC_METADATA_URL}`, `AUTH_OIDC_CLIENT_ID}`, and `{AUTH_OIDC_CLIENT_SECRET}` are attributes, which is not the case.

* As there is no user substituted value, the `quotes` substitution is unused, and not necessary.